app/Http/Controllers/Admin/DepartmentController.php

```diff
@@ -54,12 +54,22 @@ public function update(DepartmentRequest $request, $id)
     public function destroy($id)
     {
         $department = Department::findOrFail($id);
-        foreach ($department->users()->get() as $user)
-        {
+        foreach ($department->users()->get() as $user) {
             $user['department_id'] = null;
             $user->update();
```

Comment:
k nên query trong foreach nhé, có thể sử dụng `saveMany` https://laravel.com/docs/6.x/eloquent-relationships#the-save-method